docs/Using-Fleet/FAQ.md

```diff
@@ -383,3 +384,18 @@ There is a [bug](https://github.com/fleetdm/fleet/issues/8443) in MySQL validati
 Depending on your infrastructure capabilities, and the number of hosts enrolled into your Fleet instance, Fleet might be slow or unresponsive after globally enabling a feature like [software inventory](https://fleetdm.com/docs/deploying/configuration#software-inventory).
 
 In those cases, we recommend a slow rollout by partially enabling the feature by teams using the `features` key of the [teams configuration](https://fleetdm.com/docs/using-fleet/configuration-files#teams).
+
+## Why am I seeing an "unsupported key" error when updating agent options?
+
+When updating agent options, you may see an error similar to this:
+
+```
+Could not update settings. unsupported key provided: "logger_plugin"
```

Comment:
I think the error here should be something like
` Bad Request: unsupported key provided: "logger_plugin"`

Reply:
Is that the error in fleetctl ?  If so, maybe I should just include the part that's the same:

```
unsupported key provided: "logger_plugin"
```

Reply:
I agree, better to include the part that's the same

Reply:
Done! Thanks again.